### ISSUE TITLE
Row.RelativeItem(): second RelativeItem can't take more height than first

### ISSUE THREAD

**Issue by MichelMichels** [QuestPDF/QuestPDF#124]
**Describe the bug**
I am trying to use the row.RelativeItem(...) method to have 2 columns 50/50. This works perfectly except when the content of the second RelativeItem(...) is greater than the content of the first. It gives a `DocumentLayoutException`.

**To Reproduce**
Use following code:
```csharp
container.Row(row =>
{
    row.RelativeItem().Background(Colors.Red.Accent1).Column(column =>
    {
        column.Item().Text("...");
    });
    
    row.RelativeItem().Background(Colors.Blue.Accent1).Column(column =>
    {
        column.Item().Text("...");
        column.Item().Text("...");
    });
});
```

**Expected behavior**
The first RelativeItem grows in height with the second one.

**Screenshots**
I can't screenshot the behavior as it crashes before I can see the result.

**Additional context**
I looked at the docs for this behavior, but did not find it.

**Comment by MarcinZiabek** [comment]
Hello, I am getting the following result:
![image](https://user-images.githubusercontent.com/9263853/154507543-cab8393b-f8a3-44a6-865a-302f8d510409.png)

Is it different than expected?

**Comment by MichelMichels** [comment]
@MarcinZiabek we have found the culprit. The code I posted is abbreviated and what isn't showing in the second RelativeItem is a LineHorizontal element. We found that if that line is the last item, the exception is thrown.

**Comment by MarcinZiabek** [comment]
Can you please post new, minimal example that does not work. I would like to make sure we are on the same page before I continue investigation 😁

**Comment by MarcinZiabek** [comment]
Hello 😁 I have just published the 2022.2.4 release that fixes this issue. Please check if it solves your problem.

**Comment by MichelMichels** [comment]
Yes, this solves the problem! Thanks for the fast update!
